camera_system/camera_calibrator.py

```diff
@@ -50,45 +76,94 @@ def make_game_area_info(self, is_left_course: bool, game_save_path="game_course.
         """
         # ゲームエリア画像を取得
         game_area_img = self.__camera_interface.capture_frame(game_save_path)
+
         # 6色変換
         color_save_path = "color_" + game_save_path
         self.__color_changer.change_color(game_area_img, color_save_path)
 
-        # カラーIDを格納する配列を宣言
-        block_color_list = []
-        base_color_list = []
-        bonus_color = []
+        # 色IDを格納する配列を宣言
+        block_color_list = np.zeros(CameraCalibrator.__COLOR_BLOCK_NUM)
+        base_color_list = np.zeros(CameraCalibrator.__BASE_BLOCK_NUM)
+        bonus_color = np.zeros(CameraCalibrator.__BONUS_BLOCK_NUM)
```

Comment:
ゼロ埋めリストを生成したいだけなら、以下のような方法も使えます
```suggestion
        block_color_list = [0] * CameraCalibrator.__COLOR_BLOCK_NUM
        base_color_list =  [0] * CameraCalibrator.__BASE_BLOCK_NUM
        bonus_color = [0] * CameraCalibrator.__BONUS_BLOCK_NUM
```